hack/ci-check.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Depending on whether or not the current build is based on a PR or not,
+# the $TRAVIS_BRANCH variable can be inaccurate.
+# For push builds, always diff against master
+BRANCH="master"
+if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
```

Comment:
I'm thinking that if it's not a PR build, we should just always run `make ci` -  WDYT?  I'm just wondering about if we're building a tag, or one of the release branches for a patch, diffing against master may not make sense.  Either way, isn't this `if` backwards -- i.e. we want to diff against `$TRAVIS_BRANCH` if it *is* a pull request?

Reply:
Yeah, I'd agree on those points. And you're right that it was backwards.

I'm gonna simplify this a little further - in this `if` check, do make ci, and exit early. Otherwise, do the below using `$TRAVIS_BRANCH` directly. Sound good?

Reply:
Yeah, I think that makes sense